Games/types/Mafia/Meeting.js

```diff
@@ -5,6 +5,44 @@ module.exports = class MafiaMeeting extends Meeting {
     super(name, game);
   }
 
+  speak(message) {
```

Comment:
i believe this is from the GY merge commit, i undid this because of poor semantics and did the fix elsewhere

Reply:
removed